src/components/campaignguide/ScenarioStepComponent/ResolutionStepComponent.tsx

```diff
@@ -23,15 +20,11 @@ export default function ResolutionStepComponent({ step }: Props) {
   const { typography } = useContext(StyleContext);
   const { processedScenario } = useContext(ScenarioGuideContext);
   const resolution = processedScenario.scenarioGuide.resolution(step.resolution);
-  const hasDS = useSelector(hasDissonantVoices);
-  const playNarrationPressed = useCallback(() => {
-    if (resolution?.narration) {
-      playNarration(resolution.narration.id);
-    }
-  }, [resolution?.narration]);
   if (!resolution) {
     return <Text>Unknown resolution: { step.resolution }</Text>;
   }
+
+  const narration = useNarration(resolution.narration);
```

Comment:
Ahh, one of the big gotchas with hooks is you have to put them before any returns.  So you need to move this up three lines and make it `resolution?.narration`

Reply:
Ah... I'll have a look through my other hooks as well